api/src/main/java/org/infinispan/api/Experimental.java

```diff
@@ -18,8 +17,8 @@
  *
  * @since 10.0
  */
-@Retention(RUNTIME)
```

Comment:
@karesti I doubt it's of any use to have runtime retention here. So I suggest to change it. Also, would hate to see ```@Experimental``` applied to method params or local vars. Would prefer to see it applied to the enclosing method itself.